buildozer/targets/android.py

```diff
@@ -798,9 +798,22 @@ def build_package(self):
             pass
 
         # XXX found how the apk name is really built from the title
+        #gradle_files = ["build.gradle", "gradle", "gradlew"]
+        #is_gradle_build = any((
+        #    exists(join(dist_dir, x)) for x in gradle_files))
+        #if is_gradle_build:
+	__sdk_dir = self.android_sdk_dir
```

Comment:
Why creating a local `__sdk_dir` variable if it's used just once?
Also why is this double underscore syntax for a local variable?